test/unit/tile.js

```diff
@@ -224,6 +240,31 @@ describe('Tile', function () {
     });
   });
 
+  it('Google layout with custom format', function (done) {
+    const directory = fixtures.path('output.png.google.dzi');
+    rimraf(directory, function () {
+      sharp(fixtures.inputJpg)
+        .png()
+        .tile({
+          layout: 'google'
+        })
+        .toFile(directory, function (err, info) {
+          if (err) throw err;
+          assert.strictEqual('dz', info.format);
+          assert.strictEqual(2725, info.width);
+          assert.strictEqual(2225, info.height);
+          assert.strictEqual(3, info.channels);
+          assert.strictEqual('number', typeof info.size);
+          fs.stat(path.join(directory, '0', '0', '0.png'), function (err, stat) {
```

Comment:
I've also added an extreme quality setting along with a file size check (with a liberal margin of error) to verify that the format options are indeed making it through.